### DISCUSSION TITLE
Anschlussplan erledigt

### DISCUSSION THREAD

**Discussion by frank-tanne** [hoylabs/OpenDTU-OnBattery#1185] [General]
Wo finde ich den Anschlussplan für Esb32 s3 mit Display?

**Comment by schlimmchen** [comment]
Welches Display denn? Nutzt das I²C oder SPI? Es ist im Grunde "egal", wo du es anschließt. Es ist nur wichtig, dass du dann ein passendes [Device Profile](https://opendtu-onbattery.net/firmware/device_profiles/) hochlädst und auswählst. Kannst dich ja am [OpenDTU Fusion Board](https://github.com/markusdd/OpenDTUFusionDocs/blob/main/README.md) orientieren, was die Verwendung von GPIOs angeht.

**Comment by edddymax** [comment]
Habe auch keinen Schaltplan gefunden. Vielleicht hilft es  dir, wie ich es bei meinem ESP-S3 mit einem CMT2300a verbunden habe.GND -GND, GPIO2- 3, SPIO- 5, SCLK- 6, 3v3- 3v3, GPIO3- 8, FCSB- 18, CSB- 4. Geflasht mit esptool.py unter Windows.

**Comment by frank-tanne** [comment]
Ich habe es hinbekommen . Ich habe einfach im Device Profiles nachgesehen und so angeschlossen. Danke nochmal.
